FirebaseAuth/Tests/Unit/Fakes/FakeBackendRPCIssuer.swift

```diff
@@ -52,7 +52,7 @@ class FakeBackendRPCIssuer: NSObject, AuthBackendRPCIssuer {
   /** @property completeRequest
       @brief The last request to be processed by the backend.
    */
-  var completeRequest: URLRequest?
+  var completeRequest: Task<URLRequest, Never>!
```

Comment:
In the tests, it wouldn't make sense to access this property _before_ doing `_ = try? await rpcImplementation.call(with: request)`, so this is implicitly unwrapped.